Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/VisualToastNotificationBasicForm.cs

```diff
@@ -69,10 +69,11 @@ private void UpdateFadeValues()
         private void UpdateFonts()
         {
             kwlblContent.StateCommon.Font = _basicToastNotificationData.NotificationContentFont ??
-                                KryptonManager.CurrentGlobalPalette.BaseFont;
+                                            KryptonManager.CurrentGlobalPalette.BaseFont;
 
             kwlblHeader.StateCommon.Font = _basicToastNotificationData.NotificationTitleFont ??
-                               new Font(KryptonManager.CurrentGlobalPalette.BaseFont.FontFamily, 11f, FontStyle.Bold);
+                                           new Font(KryptonManager.CurrentGlobalPalette.BaseFont.FontFamily, 11f,
```

Comment:
Should use Kryptons "HeaderStyle" font, because 11 might be smaller than the Base font size !

Reply:
Now rectified

Reply:
You should be using the palette font for headers, and not creating a new one with a fixed size !

Reply:
Where is the "HeaderStyle" font stored & is there an API for retrieving it?

Reply:
Think this has now been sorted

```cs
kwlblNotificationTitle.StateCommon.Font =
                    _basicToastNotificationData.NotificationTitleFont ?? _palette.Header1ShortFont;
```